library/README.md

```diff
@@ -90,6 +91,18 @@ import stylesheet from '@kaliber/build/lib/stylesheet'
 </head>
 ```
 
+Similarly the js files are compiled to hashed names, you load them using the `javascript` library:
+
+```jsx
+import javascript from '@kaliber/build/lib/javascript'
+
+<head>
+  { javascript }
+</head>
+```
+
+If you want to use the scripts in another language or framework, use the `chunk-manifest.json` file to obtain the correct file names.
```

Comment:
Care to explain where the `chunk-manifest.json` can be found?

Reply:
Yeah, it will be in the target dir, but I will add that